app/templates/components/nav-bar.hbs

```diff
@@ -20,23 +20,23 @@
       {{#if session.isAuthenticated}}
         {{#ui-dropdown class='item' action='hide'}}
           {{widgets/safe-image isAvatar=true class='ui avatar round image nav-bar-image' src=(if authManager.currentUser.avatarUrl authManager.currentUser.avatarUrl
-                    '/images/placeholders/avatar.png')}}
+                      '/images/placeholders/avatar.png')}}
           {{if authManager.currentUser
-               (if authManager.currentUser.firstName authManager.currentUser.firstName authManager.currentUser.email)
-               'Welcome'}}
+                (if authManager.currentUser.firstName authManager.currentUser.firstName authManager.currentUser.email)
+                 'Welcome'}}
           <i class="dropdown icon"></i>
-          <div class="menu">
-            <a href="{{href-to 'my-tickets'}}" class="item">{{t 'My Tickets'}}</a>
-            <a href="{{href-to 'my-sessions'}}" class="item">{{t 'My Sessions'}}</a>
-            <a href="{{href-to 'events.list' 'live'}}" class="item">{{t 'Manage Events'}}</a>
+          <div class="menu dropdown-menu">
+            <a href="{{href-to 'my-tickets'}}" class="item user-profile-dropdown-item">{{t 'My Tickets'}}</a>
```

Comment:
Please use the existing item class to refer to it. Add a custom class if required to the parent instead.

Reply:
I feel this is more specific to the element if you want I can update that.

Reply:
@akshitahluwalia I think you mis-understood what I said ... 

What I mean is ... Instead of adding a custom class to each of the child elements, add a custom class only to the parent and access the child elements via the parent's custom class.

For example,

```html
<div class="ui dropdown">
  <div class="menu user-menu">
      <a class="item user-menu-item"></a>
      <a class="item user-menu-item"></a>
      <a class="item user-menu-item"></a>
  </div>
</div>
```

Instead doing the above, a more cleaner way of doing it would be,

```html
<div class="ui user dropdown">
  <div class="menu">
      <a class="item"></a>
      <a class="item"></a>
      <a class="item"></a>
  </div>
</div>
```

Now, in scss you can use

```scss
.ui.user.dropdown{
   /* dropdown specific code */

  .menu {
    /* Menu specific code */

    .item {
        /* item specific code */
    }
  }
}
```

Isn't this more cleaner and easier to understand ?

Reply:
Sure I didn't get what you said in the first place. Please check the changes now.